pkg/plugin/test-keeper/plugin/event_handler_gh_test.go

```diff
@@ -55,7 +58,7 @@ var _ = Describe("Test Keeper Plugin features", func() {
 
 			gock.New("https://api.github.com").
 				Post("/repos/" + repositoryName + "/statuses").
-				SetMatcher(ExpectPayload(toBe(github.StatusSuccess, keeper.TestsExistMessage, keeper.TestsExistDetailsLink))).
+				SetMatcher(ExpectPayload(toBe(github.StatusSuccess, keeper.TestsExistMessage, utils.StringJoin(botName, keeper.ProwPluginName), keeper.TestsExistDetailsLink))).
```

Comment:
how about using a const for expected context name, as this is not changing anywhere in the code?

```golang
const (
  expectedContext = utils.StringJoin(botName, keeper.ProwPluginName)
)
```